### ISSUE TITLE
Publish 5.0.x version on CocoaPods

### ISSUE THREAD

**Issue by angelolloqui** [mxcl/PromiseKit#751]
* PromiseKit version: **5.0.3**
* Xcode version: **9.0**

Versions 5.0.x are not under CocoaPods. Could you push the latest version to trunk?

https://github.com/CocoaPods/Specs/tree/master/Specs/a/f/1/PromiseKit
https://cocoapods.org/pods/PromiseKit

```
-> PromiseKit (4.4.3)
   Promises for Swift & ObjC.
   pod 'PromiseKit', '~> 4.4.3'
   - Homepage: http://promisekit.org
   - Source:   https://github.com/mxcl/PromiseKit.git
   - Versions: 4.4.3, 4.4.2, 4.4.0, 4.3.2, 4.3.1, 4.2.2, 4.2.0, 4.1.7, 4.1.4, 4.1.3, 4.1.2, 4.1.0, 4.0.5, 4.0.4, 4.0.3, 4.0.1, 4.0.0, 3.5.3, 3.5.2, 3.5.1, 3.5.0, 3.4.4, 3.4.3,
   3.4.2, 3.4.1, 3.4.0, 3.3.0, 3.2.1, 3.2.0, 3.1.1, 3.1.0, 3.0.3, 3.0.2, 3.0.1, 3.0.0, 2.2.1, 2.2.0, 2.1.3, 2.1.2, 2.1.1, 2.1.0, 2.0.6, 2.0.5, 2.0.4, 2.0.3, 2.0.2, 2.0.1, 2.0.0,
   1.7.2, 1.7.1, 1.7.0, 1.6.0, 1.5.3, 1.5.2, 1.5.1, 1.5.0, 1.4.3, 1.4.2, 1.4.1, 1.4.0, 1.3.0, 1.2.5, 1.2.4, 1.2.3, 1.2.2, 1.2, 1.0.3, 1.0.2, 1.0.1, 1.0, 0.9.21, 0.9.19, 0.9.18,
   0.9.17.1, 0.9.17, 0.9.16.6, 0.9.16.5, 0.9.16.4, 0.9.16.3, 0.9.16.2, 0.9.16.1, 0.9.16, 0.9.15.3, 0.9.15.2, 0.9.15.1, 0.9.15, 0.9.14.3, 0.9.14.2, 0.9.14.1, 0.9.14, 0.9.13.2,
   0.9.13.1, 0.9.13, 0.9.11.1, 0.9.11, 0.9.10, 0.9.9, 0.9.8.1, 0.9.8, 0.9.7.5, 0.9.7.4, 0.9.7.3, 0.9.7.2, 0.9.7.1, 0.9.7, 0.9.6, 0.9.5, 0.9.4, 0.9.3, 0.9.2, 0.9.1, 0.9.0 [master
   repo]
   - Subspecs:
     - PromiseKit/Accounts (4.4.3)
     - PromiseKit/Alamofire (4.4.3)
     - PromiseKit/AddressBook (4.4.3)
     - PromiseKit/AssetsLibrary (4.4.3)
     - PromiseKit/AVFoundation (4.4.3)
     - PromiseKit/Bolts (4.4.3)
     - PromiseKit/CloudKit (4.4.3)
     - PromiseKit/CoreBluetooth (4.4.3)
     - PromiseKit/CorePromise (4.4.3)
     - PromiseKit/CoreLocation (4.4.3)
     - PromiseKit/EventKit (4.4.3)
     - PromiseKit/Foundation (4.4.3)
     - PromiseKit/MapKit (4.4.3)
     - PromiseKit/MessageUI (4.4.3)
     - PromiseKit/OMGHTTPURLRQ (4.4.3)
     - PromiseKit/Photos (4.4.3)
     - PromiseKit/QuartzCore (4.4.3)
     - PromiseKit/Social (4.4.3)
     - PromiseKit/StoreKit (4.4.3)
     - PromiseKit/SystemConfiguration (4.4.3)
     - PromiseKit/UIKit (4.4.3)
     - PromiseKit/UIImagePickerController (4.4.3)
     - PromiseKit/WatchConnectivity (4.4.3)
```

**Comment by mxcl** [comment]
I am not ready to publish it to CocoaPods. Until then if you need it you can amend your `Podfile` too fetch ~> 5.0 via its tag.

**Comment by nolanw** [comment]
Just in case anyone else wants to try out ~> 5.0 before it gets pushed to CocoaPods and tries to include a non-default subspec: you'll need to add `submodules: true` to your `pod 'PromiseKit'` line in your `Podfile`. Otherwise CocoaPods will silently fail to download and set up any of the subspecs.

**Comment by jsflax** [comment]
What is the estimated release date for 5.0 on CocoaPods?

**Comment by mxcl** [comment]
I need to write a blog post explaining the changes, I need to rework the docs so there are branches for v4 and v5 and then update the README. So it's contingent on that. Hopefully I'll have time over the holidays.

**Comment by jsflax** [comment]
Awesome, thanks for the quick response. What's the likelihood of the current API changing before release? We'd like to make our own major update and include 5.0.0 in our own library before the holidays, and it would be contingent on the current 5.0.0 API not changing.

**Comment by mxcl** [comment]
I respect semver so there will be no breaking changes until 6

**Comment by mxcl** [comment]
Closing in favor of #760
